tests/integration/pure_cairo_vm/cheatcodes/mock_call/mock_call_test.cairo

```diff
@@ -0,0 +1,95 @@
+from starkware.cairo.common.math import assert_not_zero
```

Comment:
Add a test for checking if mocking a call for contract behind a proxy works as well.